### PR TITLE
WIP: Comment System integration in Current Event

### PR DESCRIPTION
**Pull request by rahil051**

### CODE REVIEW

File: client/src/components/CommentBox/CommentBox.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <h2>Comments</h2>
+        <h3>Please Sign In/Register to post comments</h3>
+        <CommentList comments={comments} />
+      </div>
+    );
+  }
+};
+
+const mapStateToProps = state => {
+  return state;
```
Comment (mashhoodr) [inline client/src/components/CommentBox/CommentBox.js]: @rahil051 why are you returning the complete state? We just need currentUser?

File: client/src/components/CommentBox/CommentForm/CommentForm.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class CommentForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      text: '',
```
Comment (mashhoodr) [inline client/src/components/CommentBox/CommentForm/CommentForm.js]: Why does text need to be in state?

File: client/src/components/CommentBox/CommentBox.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import CommentList from './CommentList/CommentList';
+import CommentForm from './CommentForm/CommentForm';
+import { connect } from 'react-redux';
+
+const CommentBox = props => {
+  const onCommentSubmit = e => {};
+
+  const commentsArray = [
```
Comment (mashhoodr) [inline client/src/components/CommentBox/CommentBox.js]: @rahil051 please link to API, its already merged

**Comment by mashhoodr** [comment]
@taleymirza maybe you can complete this one as well.

**Comment by mashhoodr** [comment]
@taleymirza whats the status of this

**Comment by taleymirza** [comment]
please close this PR, I have worked on a new PR for comment section

**Comment by taleymirza** [comment]
please close this PR I have restarted work for comment section in another PR.

**Pull request closed without merging**